xml/depl_crowbar.xml

```diff
@@ -0,0 +1,322 @@
+     <listitem>
+      <para>The <guimenu>Exported Files</guimenu> screen allows you to export the
+       &chef; configuration and the <literal>supportconfig</literal> TAR
+       archive. The <literal>supportconfig</literal> archive contains system
+        information such as the current kernel version being used, the hardware, RPM
+        database, partitions, and the most important log files for analysis of any
+        problems. To access the export options, click <guimenu>New
+        Export</guimenu>. After the export has been successfully finished, the
+        <guimenu>Exported Files</guimenu> screen will show any files that are
+       available for download.</para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><guimenu>Repositories</guimenu></term>
+     <listitem>
+      <para>This screen shows an overview of the configured repositories. When
+       opening this screen, the &crow; UI checks the repositories to make sure
+       that they are available and usable for each node.</para>
+      <!--<para>The User Interface above performs some checks on the repositories on
+       each request. It mainly makes sure that they are available and usable for
+       each node. You can activate the repositories by setting them as active
+       via the checkbox. That means the managed nodes will be automatically
+       configured to use repositories only if they are marked as active. The
+       repositories in this list are from a fixed list of repos, but the user
+       can manipulate this list via /etc/crowbar/repos.yml This makes it
+       possible to add new repos, but also to change the url or ask_on_error
+       setting for existing repos. and another change would be that yast-crowbar
+       doesn't allow to change the platform of a repository anymore. not sure if
+       that is relevant for the documentation.</para>-->
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><guimenu>Swift &dash;</guimenu></term>
+     <listitem>
+      <para>This screen allows you to run
+       <command>swift-dispersion-report</command> on the node or nodes
+        <remark>taroth 2016-02-03: DEVs is the following correct?</remark> to
+       which it has been deployed. Use this tool to measure the
+       overall health of the swift cluster. For details, see <link
+        xlink:href="http://docs.openstack.org/liberty/config-reference/content/object-storage-dispersion.html"
+       />. 
+       <remark>taroth 2016-02-03: DEVs, can I open the resulting report from
+        here? I tried to run a report but it seems to run forever with no result (see
+        bsc#964946)</remark></para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><guimenu>Backup &amp; Restore</guimenu></term>
+     <listitem>
+      <para>This screen lets you create a backup of the &admserv; and download
+       it. You can also restore a backup or upload a backup image from your file
```

Comment:
MINOR: maybe `You can also restore _from_ a backup` ?
and maybe `upload from your _local_ filesystem`